### ISSUE TITLE
Training loss

### ISSUE THREAD

**Issue by ito-kota** [guoqincode/Open-AnimateAnyone#109]
Hi!
Thak you for your great work!

I have tried to train 1st stage using UBC Fashion dataset, but there is no indication that the loss is converging, just oscillating as in the attached image.

I think the output image is getting better, but is this normal behavior?

![スクリーンショット 2024-01-25 121106](https://github.com/guoqincode/Open-AnimateAnyone/assets/101609761/4675f0d1-2eae-464f-aeb1-6a44552b5631)

**Comment by guoqincode** [comment]
Hello, Generative models like diffusion model Unlike regular supervised models, we can't use loss to determine "convergence" but rather visualization results.
